lib/gringotts/gateways/mercadopago.ex

```diff
@@ -151,8 +153,9 @@ defmodule Gringotts.Gateways.Mercadopago do
   # network request in here, and parse it using another private method called
   # `respond`.
   #@spec commit(_, _, _, _) :: {:ok | :error, Response}
-  defp commit(method, path, body, headers) do
-    HTTPoison.request(method, path, body, headers, [])
+  defp commit(method, path, body, opts) do
```

Comment:
1. This function doesn't actually use `opts` as of now. If you pass `opts` to the `request`, there's some benefit.
2. Could you please add the right `@spec`?